packages/type-utils/tests/TypeOrValueSpecifier.test.ts

```diff
@@ -0,0 +1,306 @@
+import { parseForESLint } from '@typescript-eslint/parser';
+import type { TSESTree } from '@typescript-eslint/utils';
+import Ajv from 'ajv';
+import path from 'path';
+
+import type { TypeOrValueSpecifier } from '../src/TypeOrValueSpecifier';
+import {
+  typeMatchesSpecifier,
+  typeOrValueSpecifierSchema,
+} from '../src/TypeOrValueSpecifier';
+
+describe('TypeOrValueSpecifier', () => {
+  describe('Schema', () => {
+    const ajv = new Ajv();
+    const validate = ajv.compile(typeOrValueSpecifierSchema);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    function runTestPositive(data: any): void {
```

Comment:
For both of these, it's almost always better to prefer `unknown` if possible:

```suggestion
    function runTestPositive(data: unknown): void {
```